packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -126,7 +126,7 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
                         ? extensions.toggleSelectWidgetMode
                         : extensions.toggleOnDeviceWidgetInspector
                   ],
-                  minIncludeTextWidth: 650,
+                  minIncludeTextWidth: 900,
```

Comment:
put 900 in a const in this class and use it everywhere 900 is used, including the expand collapse buttons